domain/src/main/java/uk/gov/hmcts/ccd/definition/store/domain/validation/displaygroup/DisplayGroupInvalidShowConditionFieldForTab.java

```diff
@@ -0,0 +1,50 @@
+package uk.gov.hmcts.ccd.definition.store.domain.validation.displaygroup;
+
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationError;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationErrorMessageCreator;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.DisplayGroupEntity;
+
+public class DisplayGroupInvalidShowConditionFieldForTab extends ValidationError {
+
+    private String showConditionField;
+    private DisplayGroupEntity displayGroup;
+
+    public DisplayGroupInvalidShowConditionFieldForTab(String showConditionField, DisplayGroupEntity displayGroup) {
+        super(
+                String.format(
+                        "Invalid show condition '%s' for display group '%s': unknown field '%s'",
```

Comment:
wonder whether we should just call display gruoup as tab in user facing error messages